Makefile

```diff
@@ -64,7 +64,7 @@ docker-hub-push: docker
 	docker push $(DOCKER_HUB_REPO):$(VERSION)
 
 run-osx: dep darwin
-	bin/prometheus-msteams-darwin-amd64 server $(RUN_ARGS)
+	bin/prometheus-msteams-darwin-amd6 -http-addr localhost:2000 $(RUN_ARGS)
```

Comment:
Typo in `amd6` here. By the way, would it be better if this is just `go run cmd/server/main.go` instead?

Reply:
@bzon I changed it to a target named `local` which seems to make more sense to me.